### PR TITLE
Bumping OPM version from v1.19.1 to v1.21.0 (one of its motivations is to provide support for ppc64le and s390x arches)

### PR DESCRIPTION
**Pull request by aemperador**
**Description of the change:**
Bump OPM version to 1.21.0 from 1.19.1 to support ppc64le and s390x arches

**Motivation for the change:**
Need support for ppc64le and s390x for official support

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [X] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))

### CODE REVIEW

**Comment by gquillar** [comment]
Change in testdata/ansible/memcached-operator/Makefile is missing

File: changelog/fragments/bump-opm-version.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Bumping OPM version from v1.19.1 to v1.21.0 to allow for ppc64le and s390x support
```
Comment (camilamacedo86) [inline changelog/fragments/bump-opm-version.yaml:5]:
```suggestion
      Bumping OPM version from v1.19.1 to v1.21.0. For further information about this changes see: https://github.com/operator-framework/operator-registry/releases
```

Comment (camilamacedo86) [reply changelog/fragments/bump-opm-version.yaml:5]: @gquillar, 

Thank you for the contribution 🥇 

Can we also update the title? 
Bumping OPM version from v1.19.1 to v1.21.0 (one of its motivations is to provide support for  ppc64le and s390x arches)

Comment (aemperador) [reply changelog/fragments/bump-opm-version.yaml:5]: I've updated the title. I will also commit your suggestion.

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

Thank you for the nice contribution 🥇

**Comment by camilamacedo86** [comment]
Closing this one since master branch have now a OPM version upper than that 
Thank you for your collab 🥇

**Pull request closed without merging**